### ISSUE TITLE
getOidcUser throws TypeError after removeOidcUser

### ISSUE THREAD

**Issue by aido179** [perarnborg/vuex-oidc#107]
Pretty sure this is related to issue #99  
 
Reproduction:
 ```
mounted: async function () {
    await this.getOidcUser() // no error
    this.removeOidcUser()
    await this.getOidcUser() // throws error
}
```

There is no chaining check in getOidcUser so if ```user``` is null, ```user.profile``` throws an error.
In your reply to issue #99, you ask what should be done - I would say if user is null, getOidcUser should throw an error that explains that there is no user to get. Alternatively,  return null, false, an empty user object.

**Comment by perarnborg** [comment]
Perfect, good input! I will fix this so that getOidcUser returns null if there is no user. To me that feels better than throwing an error.

**Comment by perarnborg** [comment]
Fixed as of v3.5.2
